README.rst

```diff
@@ -0,0 +1,35 @@
+# Cyclus Organization
+
+This organization encompasses the repositories 
+of and relating to the Cyclus fuel cycle 
+simulator code. The three most-used repositories
+are:
+* Cyclus
+* Cycamore
+* Cymetric 
+
```

Comment:
It seems like we should say more about what Cyclus is/does here... We don't want to be too redundant with fuelcycle.org, but at least a little more context?

Reply:
I can add in some more context.